pkg/cmdutil/userio/styles.go

```diff
@@ -42,19 +42,19 @@ func newStyles(renderer *lipgloss.Renderer) *styles {
 }
 
 type nonInteractiveStyles struct {
-	infoStyle        lipgloss.Style
-	warnHeadingStyle lipgloss.Style
-	warnMessageStyle lipgloss.Style
-	bold             lipgloss.Style
-	status           wizard.TaskStatusStyle
+	InfoStyle        lipgloss.Style
```

Comment:
Should we make the struct also public, since the constructor func and all the fields are also public?

Reply:
we don't technically need to since you can get it from `userio.NewNonInteractiveStyles()`

Reply:
It just feels very wrong :D
You return to the outer world unexported type and say that all of it fields are exported for some reason
It's kind of minor, but I think it brings some confusion. I wouldn't expect to see private type usages in other packages

Reply:
updated :+1: